packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -188,7 +187,7 @@ export function getInternalReactConstants(
       LazyComponent: 16,
       MemoComponent: 14,
       Mode: 8,
-      OffscreenComponent: 23, // Experimental
+      OffscreenComponent: 22, // Experimental
```

Comment:
This change looks wrong. `OffscreenComponent` is `22` in master now, but in 17.0.0 and 17.0.1 it's `23`.

Reply:
Fixed in #20362